pyvista/plotting/widgets.py

```diff
@@ -884,18 +884,15 @@ def normalize(point, viewport):
             if not isinstance(style, str):
                 raise TypeError("Expected type for ``style`` is str but"
                                 f" {type(style)} was given.")
-            style_params = rcParams['slider_style'].get(style, None)
-            if style_params is None:
-                raise KeyError("The requested style does not exist: "
-                               f"{style}. The styles available are {list(rcParams['slider_style'].keys())}.")
-            slider_rep.SetSliderLength(style_params['slider_length'])
-            slider_rep.SetSliderWidth(style_params['slider_width'])
-            slider_rep.GetSliderProperty().SetColor(style_params['slider_color'])
-            slider_rep.SetTubeWidth(style_params['tube_width'])
-            slider_rep.GetTubeProperty().SetColor(style_params['tube_color'])
-            slider_rep.GetCapProperty().SetOpacity(style_params['cap_opacity'])
-            slider_rep.SetEndCapLength(style_params['cap_length'])
-            slider_rep.SetEndCapWidth(style_params['cap_width'])
+            slider_style = getattr(pyvista.global_theme.slider_style, style)
```

Comment:
This makes me think that we should rename `global_theme.slider_style` to `global_theme.slider_styles`.

Reply:
Agreed.